.github/workflows/apt.yml

```diff
@@ -235,24 +235,28 @@ jobs:
       - uses: actions/download-artifact@v4
         with:
           name: packages-apt-source-${{ matrix.rake-job }}
-      - uses: canonical/setup-lxd@v0.1.2
-      - name: Run diagnostic
+      - name: Setup Incus
         run: |
-          uname -a
-          echo "::group::snap info lxd"
-          snap info lxd
-          echo "::endgroup::"
-          echo "::group::snap services lxd"
-          snap services lxd
-          echo "::endgroup::"
-          echo "::group::snap logs lxd"
-          sudo snap logs lxd
-          echo "::endgroup::"
-          echo "::group::lxc remote list"
-          lxc remote list
-          echo "::endgroup::"
-          echo "::group::lxc list images:"
-          lxc image list images:
-          echo "::endgroup::"
-      - name: Run Test  ${{ matrix.test }} on ${{ matrix.lxc-image }}
-        run: fluent-package/apt/systemd-test/test.sh ${{ matrix.lxc-image }} ${{ matrix.test }}
+          sudo curl -fsSL https://pkgs.zabbly.com/key.asc -o /etc/apt/keyrings/zabbly.asc
```

Comment:
If we use ubuntu 24.04, incus include Ubuntu official repository.
So, we don't use pkgs.zabbly.com.

See:
https://github.com/mroonga/mroonga/blob/main/.github/workflows/linux.yml#L275-L278
https://packages.ubuntu.com/noble/incus

Reply:
Oh, thanks!